client-metrics-reporter/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.strimzi</groupId>
+        <artifactId>metrics-reporter</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>client-metrics-reporter</artifactId>
+
+    <dependencies>
+        <dependency>
```

Comment:
For multi module projects, maven best practice is to express shared dependencies in the parent POM using a dependencyManagement section.  You can then remove the version numbers from the dependencies in the child poms.

Reply:
Thanks for the tip! I already had the versions defined in the parent POM so I removed the versions from the modules.